### PR TITLE
Update distribution.yaml - Add franka_ros2

### PR DESCRIPTION
**Pull request by Smith-JackSmith**
# Please Add This Package to be indexed in the rosdistro.

franka_ros2

# The source is here:

https://github.com/frankaemika/franka_ros2

# Checks
 - [ x] All packages have a declared license in the package.xml
 - [ x] This repository has a LICENSE file
 - [ x] This package is expected to build on the submitted rosdistro

### CODE REVIEW

File: humble/distribution.yaml [hunk 1]
```diff
@@ -2641,6 +2641,12 @@ repositories:
       url: https://github.com/frankaemika/franka_description.git
       version: main
     status: developed
+  franka_ros2:
+    source:
+      type: git
+      url: https://github.com/frankaemika/franka_ros2
```
Comment (clalancette) [inline humble/distribution.yaml:2647]:
```suggestion
      url: https://github.com/frankaemika/franka_ros2.git
```

**Review by clalancette — DISMISSED** [review]

**Review by mjcarroll — APPROVED** [review]

**Comment by Smith-JackSmith** [comment]
What is  the status of this PR ?

**Comment by mjcarroll** [comment]
> What is the status of this PR ?

It was merged about an hour ago.  You should now appear in the source index for the ROS packages, but won't have documentation or binary package releases.

**Comment by Smith-JackSmith** [comment]
Thank you. 

I find this status page (if that's the correct name) to be a bit messy.

Thank you for clarifying.

**Pull request merged**